oer-bookr/src/components/review/Review.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+import { ReviewWrapper } from "../../styles/reviewStyles"
+
+const Review = props => {
+  return (
+    <ReviewWrapper>
+      <div>
+        <strong>{props.rev.reviewer}</strong>
+      </div>
+      <div>{props.rev.review}</div>
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) to make the repeated dot chaining a little shorter:
```javascript
// this puts the value of rev off of the props object that's passed in 
// into a variable of the same name
const Review = ({ rev }) => {
  return (
    <ReviewWrapper>
      <div>
        <strong>{rev.reviewer}</strong>
      </div>
      <div>{rev.review}</div>
    </ReviewWrapper>
  )
}
```

Reply:
Thanks for all the good feedback!